deepcli/deepcli/cli.py

```
@@ -218,7 +218,16 @@ def cmd_export(args):
     fmt = args.format or "json"
     output = export_json(token, sid) if fmt == "json" else export_markdown(token, sid)
     if args.output:
-        Path(args.output).write_text(output)
+        out_path = Path(args.output)
+        if out_path.is_symlink():
+            console.print(f"[red]Export target '{args.output}' is a symlink. Rejected for security.[/]")
```

Pylint (4.0.7)

[convention] Line too long (104/100)

(C0301)